review_management.md

```diff
@@ -37,6 +37,38 @@ Cursory review is meant to be a rapid and lightweight "triage" process which kee
+
+Finally, evolution proposals are expected to build on each other. When a proposal has been accepted but is not yet available in the language, it puts related proposals in the awkward position of needing to accommodate a feature that doesn't really exist. The evolution process should not build castles in the sky; there should be a feasible path for delivering every accepted feature within a reasonable period of time, preventing accepted proposals from indefinitely holding up other work.
+
+That said, the provided implementation doesn't need to be ready to commit as soon as the review is complete. It may not even be the implementation that code maintainers will ultimately accept. The initial proposal implementation just needs to be sufficient to satisfy the three goals above:
+
+- The implementation must fully implement the proposed feature. If part of the feature is proving harder to implement, that may indicate significant theoretical or practical problems; an incomplete implementation should not be ignored as something that can be finished later.
+
+- The implementation must take a sound conceptual approach as judged by code maintainers in the affected areas. It may have significant implementation flaws, such as not fitting into the compiler architecture correctly, but the basic idea of the implementation should not be dodging major theoretical concerns. Cursory reviewers should engage with appropriate code maintainers in order to get their opinion about whether the implementation satisfies this point.
+
+- The implementation needs to work well enough for its consequences to be tested, both by Swift users and by automated systems. All of the required steps here are achievable automatically with the [Swift CI system](https://www.swift.org/continuous-integration/).
+
+  - At a minimum, a PR enabling the feature must pass the full Swift project correctness test suite.
```

Comment:
By “the full Swift project correctness test suite”, do you mean the tests run by “swift-ci please test", or do you mean something else?

Reply:
Yes, exactly.  Clarified.